### PR TITLE
Coloring connectomics

### PR DESCRIPTION
**Pull request by frheault**
Modifying existing scripts to facilitate Laurent's life with coloring bundles when doing connectomics-like approach.

The Uniformize endpoints scripts now accept ROIs (similar to the --drawn_roi X.nii.gz 1:5) to perform a head/tail flip based on the proximity to the ROI's barycenter (rather than an x/y/z method).

Then a new coloring option was added to color along the length using default OR custom colormap (declared as `--colormap red-orange-yellow-green-blue-pink`)

To be tested by Laurent first.

### CODE REVIEW

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

File: scilpy/utils/streamlines.py [hunk 1]
```diff
@@ -119,6 +120,58 @@ def uniformize_bundle_sft(sft, axis=None, swap=False):
...
+    sft: StatefulTractogram
+         The tractogram that contains the list of streamlines to be uniformized
+    mask: np.ndarray
+        Mask to use as a reference for the barycenter.
+    swap: boolean, optional
+        Swap the orientation of streamlines
+    """
+
+    barycenter = np.average(np.argwhere(mask), axis=0)
+    print(barycenter)
```
Comment (AntoineTheb) [inline scilpy/utils/streamlines.py:137]: leftover print ?

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

**Comment by arnaudbore** [comment]
@frheault conflict + any news from Laurent ?

**Comment by arnaudbore** [comment]
Build Failed :boom: 
https://jenkins.scil.usherbrooke.ca/job/scilpy/job/PR-636/6/

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

**Pull request merged**